pkg/util/mount/mount.go

```diff
@@ -199,3 +199,32 @@ func getDeviceNameFromMount(mounter Interface, mountPath, pluginDir string) (str
 
 	return path.Base(mountPath), nil
 }
+
+// IsNotMountPoint determines if a directory is a mountpoint.
+// It should return ErrNotExist when the directory does not exist.
+// This method uses the List() of all mountpoints
+// It is more extensive than IsLikelyNotMountPoint
+func IsNotMountPoint(mounter Interface, file string) (bool, error) {
+	notMnt, notMntErr := mounter.IsLikelyNotMountPoint(file)
+	if notMntErr != nil {
+		return notMnt, notMntErr
+	}
+	// identified as mountpoint, so return this fact
+	if notMnt == false {
+		return notMnt, nil
+	}
+	// check all mountpoints since IsLikelyNotMountPoint
+	// is not reliable for some mountpoint types
+	mountPoints, mountPointsErr := mounter.List()
+	if mountPointsErr != nil {
+		return notMnt, mountPointsErr
+	}
+	deletedFile := fmt.Sprintf("%s\\040(deleted)", file)
```

Comment:
This parsing is implementation specific. It should be done in the the mounter implementation instead.